src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlSymmetricKeyCache.cs

```diff
@@ -55,35 +55,20 @@ internal bool GetKey(SqlEncryptionKeyInfo keyInfo, out SqlClientSymmetricKey enc
 #endif //DEBUG
 
             // Lookup the key in cache
-            encryptionKey = _cache.Get(cacheLookupKey) as SqlClientSymmetricKey;
+            SqlClientSymmetricKey encryptionKey = _cache.Get(cacheLookupKey) as SqlClientSymmetricKey;
```

Comment:
Please use pattern matching to avoid [IDE0019](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0019). 
basically you can take this line out and define the value inside the `if`'
something like 
`if (!(_cache.Get(cacheLookupKey) is SqlClientSymmetricKey encryptionKey))`